src/Development/Shake/Command.hs

```diff
@@ -154,6 +134,8 @@ commandExplicit funcName copts results exe args = do
                             , any (`isPrefixOf` x) inside
                             , not $ any ($ x) ignore]
                 rel = map (makeRelative cwd)
+            when (Autodeps `elem` copts) $
+              needed $ rel $ ham rs
```

Comment:
Damn, I added a .dir-locals.el just for that, but it's on another machine :-(
Maybe we could add one to the repo with your preferred settings?


> On 03 Nov 2015, at 23:26, Neil Mitchell <notifications@github.com> wrote:
> 
> In src/Development/Shake/Command.hs:
> 
> > @@ -154,6 +134,8 @@ commandExplicit funcName copts results exe args = do
> >                              , any (`isPrefixOf` x) inside
> >                              , not $ any ($ x) ignore]
> >                  rel = map (makeRelative cwd)
> > +            when (Autodeps `elem` copts) $
> > +              needed $ rel $ ham rs
> 
> 4 space indents are preferred, although not really a big deal (but I would reindent it if I'm ever near that line).
> 
> —
> Reply to this email directly or view it on GitHub.
>